docs/whats-new/2020-10.md

```diff
@@ -0,0 +1,258 @@
+- [SYSLIB0003: Code access security is not supported](../core/compatibility/syslib0003) - SYSLIB reference
+- [SYSLIB0004: The constrained execution region (CER) feature is not supported](../core/compatibility/syslib0004)
+  - Update syslib0004 docs with workarounds
+  - SYSLIB reference
+- [SYSLIB0005: The global assembly cache (GAC) is not supported](../core/compatibility/syslib0005) - SYSLIB reference
+- [SYSLIB0006: Thread.Abort is not supported](../core/compatibility/syslib0006) - SYSLIB reference
+- [SYSLIB0007: Default implementations of cryptography algorithms not supported](../core/compatibility/syslib0007) - SYSLIB reference
+- [SYSLIB0008: CreatePdbGenerator is not supported](../core/compatibility/syslib0008) - SYSLIB reference
+- [SYSLIB0009: The AuthenticationManager Authenticate and PreAuthenticate methods are not supported](../core/compatibility/syslib0009) - SYSLIB reference
+- [SYSLIB0010: Unsupported remoting APIs](../core/compatibility/syslib0010) - SYSLIB reference
+- [SYSLIB0011: BinaryFormatter serialization is obsolete](../core/compatibility/syslib0011) - SYSLIB reference
+- [SYSLIB0012: Assembly.CodeBase and Assembly.EscapedCodeBase are obsolete](../core/compatibility/syslib0012) - SYSLIB reference
+- [ReadyToRun Compilation](../core/deploying/ready-to-run) - Document PublishReadyToRun
+- [Tracing .NET applications with PerfCollect](../core/diagnostics/trace-perfcollect-lttng) - Add tutorial on collecting traces with PerfCollect
+- [Console log formatting](../core/extensions/console-log-formatter) - Adding console log formatting article
+- [Releases and support for .NET Core and .NET 5](../core/releases-and-support) - Releases and support doc
+- [dotnet nuget verify](../core/tools/dotnet-nuget-verify) - dotnet nuget verify doc
+- [NETSDK1005 and NETSDK1047: Asset file is missing target](../core/tools/sdk-errors/netsdk1005) - Adding .NET SDK error pages for 1005, 1047, and 1059
+- [NETSDK1013: The TargetFramework value was not recognized](../core/tools/sdk-errors/netsdk1013) - Create netsdk1013.md
+- [NETSDK1022: Duplicate items were included.](../core/tools/sdk-errors/netsdk1022) - Create netsdk1022.md
+- [NETSDK1059: Project contains obsolete .NET CLI tool](../core/tools/sdk-errors/netsdk1059) - Adding .NET SDK error pages for 1005, 1047, and 1059
+- [NETSDK1071: Explicitly versioned PackageReference to a metapackage that would be included with the framework.](../core/tools/sdk-errors/netsdk1071) - Create a page for NETSDK1071
+- [NETSDK1073: The FrameworkReference was not recognized](../core/tools/sdk-errors/netsdk1073) - Create netsdk1073.md
+- [NETSDK1079: The Microsoft.AspNetCore.All package is not supported when targeting .NET Core 3.0 or higher.](../core/tools/sdk-errors/netsdk1079) - Add error message documentation for NETSDK1079
+- [Resolve NETSDK1145, targeting or apphost pack missing](../core/tools/sdk-errors/netsdk1145) - Add targeting/runtime pack missing
```

Comment:
I suggest we remove these error pages, as they're only of interest when you actually get the particular error message.